src/common/signalk/signalk.json

```diff
@@ -36,6 +36,54 @@
       "units": "K",
       "description": "Current water temperature"
     },
+    {
+      "path": "environment.wind",
+      "type": "numberValue",
+      "units": "rad",
+      "description": "Wind Data"
+    },
+    {
+      "path": "environment.wind.angleApparent",
+      "type": "numberValue",
+      "units": "rad",
+      "description": "AWA, Apparent Wind Angle"
+    },
+    {
+      "path": "environment.wind.angleTrue",
```

Comment:
`.angleTrue` [does not exist in SignalK](https://github.com/SignalK/specification/blob/master/schemas/groups/environment.json#L304). They have `angleTrueGround` and `angleTrueWater` which I think is good because it makes the difference between true wind calculated from the boat speed or true wind calculated from ground speed.

For speed they have `speedTrue` and `speedOverGround`. The descriptions are explicit about how it should be calculated.

For the description I just copy/paste the same description that is found in SignalK.

I had already added a large batch of wind variables that I just merged in `develop` which is why we now have a merge conflict. I am going to clean that up to make it easy for you to merge.